### PR TITLE
Update recipe from grayskull

### PR DESCRIPTION
**Pull request by xylar**
Add @xylar as a maintainer

Importantly, this merge adds a constraint on `setuptools`.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
I'm seeing pip-check failures in https://github.com/conda-forge/parsl-feedstock/pull/15 because of the lack of a constraint on `setuptools>=62.4.0` here.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,47 +1,52 @@
...
+    - LICENSE.ASF-2
+    - LICENSE.GPL-3
+  dev_url: https://pagure.io/python-daemon/
 
 extra:
   recipe-maintainers:
     - lincrosenbach
     - kfasolin
     - ocefpaf
+    - xylar
```
Comment (xylar) [inline recipe/meta.yaml:52]: Please let me know if you prefer I not add myself as a maintainer.  I'm happy to remove myself.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Thanks @ocefpaf!

**Comment by xylar** [comment]
@ocefpaf, since my my update to the license caused a rerender, you will need to add the label again.

**Comment by xylar** [comment]
@ocefpaf, could you please merge this at your convenience?

**Comment by xylar** [comment]
Thanks agian!

**Comment by ocefpaf** [comment]
Thanks @xylar for helping out with another feedstock.

**Pull request merged**